tensorbay/opendataset/_utility/voc.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""_get_data function for VOC2012Detection dataset and UrbanObjectionDetection dataset."""
+
+import os
+from typing import Set
+
+from tensorbay.dataset import Data
+from tensorbay.label import LabeledBox2D
+
+try:
+    import xmltodict
+except ModuleNotFoundError:
+    from tensorbay.opendataset._utility.mocker import xmltodict  # pylint:disable=ungrouped-imports
+
+
+def _get_data(
+    filename: str, image_path: str, annotation_path: str, boolean_attributes: Set[str]
+) -> Data:
+    """Get all information of the datum corresponding to filename.
+
+    Arguments:
+        filename: The filename of the data.
```

Comment:
this `filename` means the filename without extension? that mean you need an another variable name rather than `filename`.
In python `pathlib`, the word `stem` is used to represent filename without extension. It maybe a better choice.